src/diff/children.js

```diff
@@ -239,39 +275,15 @@ export function diffChildren(
  * Flatten and loop through the children of a virtual node
  * @param {import('../index').ComponentChildren} children The unflattened
  * children of a virtual node
- * @param {(vnode: import('../internal').VNode) => import('../internal').VNode} [callback]
- * A function to invoke for each child before it is added to the flattened list.
  * @param {Array<import('../internal').VNode | string | number>} [flattened] An flat array of children to modify
  * @returns {import('../internal').VNode[]}
  */
-export function toChildArray(children, callback, flattened) {
-	if (flattened == null) flattened = [];
-
+export function toChildArray(children) {
```

Comment:
Since this function is no longer used in the diff hot path, I rewrote it to optimize for size instead of perf/memory. Let me know what you think. The rewrite saved us about 40 bytes